v0/destinations/mp/transform.js

```diff
@@ -15,14 +15,21 @@ function getEventTime(message) {
   return new Date(message.originalTimestamp).toISOString();
 }
 
-function responseBuilderSimple(parameters, message, eventType) {
+function responseBuilderSimple(parameters, message, eventType, destConfig) {
   let endpoint = "https://api.mixpanel.com/engage/";
+  if (destConfig.dataResidency === "eu") {
```

Comment:
can we use ternary operator instead of overriding with a condition? I feel its more readable.